regen-network/groups-ui
# feat: add execute proposal to proposal page

Closes: #130

### How to test

You may need to test locally and update the code to allow a shorter voting period unless you have a group with an accepted proposal ready to execute.

- Create group and policy with 30 second voting period
- Create proposal (any proposal, can be text proposal)
- Vote on proposal and wait for voting period to end
- View proposal page and notice execute footer button
- Execute the proposal using the execute footer button
